network/soroban-rpc/admin-guide.mdx

```diff
@@ -59,15 +59,18 @@ curl "http://localhost:8000"
 You can interact with this local node using the Soroban CLI. First, add it as a configured network:
 
 ```bash
-soroban config network add standalone \
+soroban network add standalone \
     --rpc-url "http://localhost:8000/soroban/rpc" \
     --network-passphrase "Standalone Network ; February 2017"
 ```
 
 Then generate a unique identity (public/private keypair) to use with it:
 
 ```bash
-soroban keys generate alice
+soroban keys generate alice \
+    --rpc-url "http://localhost:8000/soroban/rpc" \
+    --network-passphrase "Standalone Network ; February 2017" \
+    --network standalone
```

Comment:
I might be wrong, but I think `--network` conflicts with the other two flags/parameters. Let's keep `--network` and remove `--rpc-url` and `--network-passphrase`, since we're trying to demonstrate the use of the network we just configured.